api/migrations/0042_add_embedded_media.py

```diff
@@ -0,0 +1,16 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0041_apply_user_enum_for_person"),
```

Comment:
We now have a couple of other migrations, which means, that this line needs to be updated